### DISCUSSION TITLE
Are RPR settings exposed yet?

### DISCUSSION THREAD

**Discussion by matiascodesal** [GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon#26] [Q&A] [answered]
I didn't see any render settings for the RPR renderer or maybe didn't know where to look.  I think as a result I'm getting banding like this:
![image](https://user-images.githubusercontent.com/8684094/102558814-4f090880-409c-11eb-869b-4725afabe6f3.png)

**Comment by bnagirniak** [comment, marked as answer]
RPR render settings aren't exposed yet, it's in our nearest tasks list.

**Comment by bsavery** [comment]
Maybe a followup question is if this should be done "Houdini Solaris" style in a node or not?

**Comment by matiascodesal** [comment]
I'm coming from Maya, so I may need to have some Blender assumptions corrected, but the way I see it is:

1. A node would be appealing from a pipeline perspective because that becomes an asset that could be shared between blend files, right?  I could have a node tree that reads in shot data with some preferred render settings setup and use that in multiple shot blend files?
2. The other use case that I see is for the user that doesn't care about USD, but just wants to use this plugin as a way to access other renderers.  They'd probably want to bypass the node tree and just use the "Scene" data source and have the render settings listed in the render properties.
